### ISSUE TITLE
Identify what has to be removed from the Doxygen system

### ISSUE THREAD

**Issue by vhuber** [wkyoshe/stageM1#14]
See http://www.feelpp.org/docs/develop/index.html
- Create a fork of feelpp
- Clean the documentation to keep only developers documentaion
- be sure every removed documentation is on book.feelpp.org

**Comment by wkyoshe** [comment]
@vhuber  The link provided above does not work.

**Comment by vhuber** [comment]
@wkyoshe Yes, you are right. That exercise will be a little bit different that exposed.
Have you forked feelpp somewhere ? Once it is done, give me an access to that forked repo.

**Comment by wkyoshe** [comment]
@vhuber it is done.

**Comment by vhuber** [comment]
Ok. I gave you the doxyfile.in and others requirements to generate the website.
Please set the mechanisms needed to let the user use `make doxygen`
Once that will be done, we will upgrade/correct it together.

**Comment by wkyoshe** [comment]
@vhuber  ok.
